### PR TITLE
Update title as optional in stageViewParams

### PR DESCRIPTION
**Pull request by Ella-ly**
For more information about how to contribute to this repo, visit [this page](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md).

## Description

> Summarize the changes, including the goals and reasons for this change. Be sure to call out any technical or behavior changes that reviewers should be aware of.

> If this Pull Request should close/resolve any issues when merged, use [the special syntax for that](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) here.

### Main changes in the PR:

1. Update `title` as optional in `stageViewParams` to match the deep link contract.

### CODE REVIEW

File: packages/teams-js/src/public/stageView.ts [hunk 1]
```diff
@@ -29,9 +29,9 @@ export namespace stageView {
     threadId: string;
 
     /**
-     * The title to give the stage view.
+     * The title to give the stage view. Test
```
Comment (TrevorJoelHarris) [inline packages/teams-js/src/public/stageView.ts:32]:
```suggestion
     * The title to give the stage view.
```

File: change/@microsoft-teams-js-521260e3-aa09-4bcd-a9c9-3f0cabcc51a4.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Update title as optional in stageViewParams",
```
Comment (TrevorJoelHarris) [inline change/@microsoft-teams-js-521260e3-aa09-4bcd-a9c9-3f0cabcc51a4.json:3]:
```suggestion
  "comment": "Made title optional when calling `stageView.open`",
```

These messages are intended for external developers who are building apps, so we want the message to accurately and succinctly capture what the change means for those developers. Does this suggestion sound correct?

**Review by TrevorJoelHarris — CHANGES_REQUESTED** [review]
Just wanted to check: did you talk with PM about this change?

File: packages/teams-js/src/public/stageView.ts (hunk 1 above)
Comment (Ella-ly) [reply packages/teams-js/src/public/stageView.ts:32]: This is for testing the e2e tests with the latest host sdk changes, I will clear it after they reverted code success.

File: change/@microsoft-teams-js-521260e3-aa09-4bcd-a9c9-3f0cabcc51a4.json (hunk 2 above)
Comment (Ella-ly) [reply change/@microsoft-teams-js-521260e3-aa09-4bcd-a9c9-3f0cabcc51a4.json:3]: Yes, thanks!

**Comment by Ella-ly** [comment]
> Just wanted to check: did you talk with PM about this change?

Yes, thanks for checking, added you in the group chat.

**Review by TrevorJoelHarris — APPROVED** [review]

**Review by KangxuanYe — APPROVED** [review]

**Pull request merged**